### PR TITLE
[ansible] Fix some issues related to ansible 2.8.7 part 3

### PR DESCRIPTION
**Pull request by msosyak**
### Description of PR

This is the continuation of #1235 and #1261 

Faced some issues when tried to run tests from the latest docker-sonic-mgmt container.

- with_dict incorrect usage
- acl.pytest failed with `AttributeError: can't set attribute`
- continuous_link_flap failed with `'interface' is undefined` and `Not all interfaces are up`

References:
https://github.com/ansible/ansible/blob/4efb8bdc4fc8b2395d0e0c5a86f6361afb697382/lib/ansible/vars/manager.py#L136
https://docs.python.org/2.7/library/functions.html#property

Summary:
Fix some issues related to ansible 2.8.7 part 3

Fixes # (issue)
Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [] Testbed and Framework(new/improvement)
- [] Test case(new/improvement)

#### How did you do it?
- fixed with_dict syntax
- changed directly `_extra_vars` instead of read-only property `extra_vars`
- uncomment part of continuous_link_flap.yml added delay for processes and interfaces to be stable

#### How did you verify/test it?
Run appropriate test

### CODE REVIEW

File: ansible/roles/test/tasks/continuous_link_flap.yml [hunk 1]
```diff
@@ -134,9 +134,9 @@
 
     - name: include continuous_peer_link_flap_helper.yml
       include_tasks: continuous_link_flap/continuous_peer_link_flap_helper.yml
-      # vars:
-      #   interface: "{{item}}"
-      # with_items: "{{ device_conn.keys() }}"
+      vars:
+        interface: "{{item}}"
+      with_items: "{{ device_conn.keys() }}"
```
Comment (msosyak) [inline ansible/roles/test/tasks/continuous_link_flap.yml]: @praveen-li @qiluo-msft  what was the reason to comment it out in initial commit?
https://github.com/Azure/sonic-mgmt/commit/45df992d4ba67239acc606ba39e8c5b2d5c35c2f#diff-82480ca497298eac7087aac65aa1b64fR127

**Comment by msosyak** [comment]
@iguohan Please review

File: ansible/roles/test/tasks/continuous_link_flap.yml (hunk 1 above)
Comment (praveen-li) [reply ansible/roles/test/tasks/continuous_link_flap.yml]: It does not seem intentional to me, I see this change in my 3rd commit (#810) but do not see any comment related to it.

File: tests/acl/test_acl.py [hunk 2]
```diff
@@ -156,7 +156,7 @@ def acl_table_config(duthost, setup, stage):
     }
 
     logger.info('extra variables for ACL table:\n{}'.format(pprint.pformat(acl_table_vars)))
-    duthost.host.options['variable_manager'].extra_vars = acl_table_vars
+    duthost.host.options['variable_manager']._extra_vars = acl_table_vars
```
Comment (lguohan) [inline tests/acl/test_acl.py]: for this one, wonder if it is better to use extra_vars.clear() and then extra_vars.update(acl_table_vars) is better? feel _extra_vars is a private object. better to operate on extra_vars object directly.

Comment (lguohan) [reply tests/acl/test_acl.py]: I see many other similiar cases, should we change all of them?

```
tests/acl/test_acl.py:    duthost.host.options['variable_manager'].extra_vars = acl_table_vars
tests/test_vrf.py:    extra_vars = {'cfg_t0': cfg_t0,
tests/test_vrf.py:    duthost.host.options['variable_manager'].extra_vars = extra_vars
tests/test_vrf.py:    extra_vars = {
tests/test_vrf.py:    ptfhost.host.options['variable_manager'].extra_vars = extra_vars
tests/test_vrf.py:    extra_vars = {
tests/test_vrf.py:    ptfhost.host.options['variable_manager'].extra_vars = extra_vars
tests/test_vrf.py:        extra_vars = {
tests/test_vrf.py:        duthost.host.options['variable_manager'].extra_vars = extra_vars
tests/test_vrf.py:        extra_vars = {
tests/test_vrf.py:        ptfhost.host.options['variable_manager'].extra_vars = extra_vars
tests/test_vrf.py:        dut_extra_vars = {
tests/test_vrf.py:        duthost.host.options['variable_manager'].extra_vars = dut_extra_vars
tests/test_vrf.py:        ptf_extra_vars = {
tests/test_vrf.py:        ptfhost.host.options['variable_manager'].extra_vars = ptf_extra_vars
tests/test_vrf.py:        dut_extra_vars.update({
tests/test_vrf.py:        duthost.host.options['variable_manager'].extra_vars = dut_extra_vars
tests/test_vrf.py:            dut_extra_vars['op_code'] = 'del'
tests/test_vrf.py:            duthost.host.options['variable_manager'].extra_vars = dut_extra_vars
tests/test_vrf.py:        dut_extra_vars = {
tests/test_vrf.py:        duthost.host.options['variable_manager'].extra_vars = dut_extra_vars
tests/test_vrf_attr.py:        extra_vars = { 'router_mac': self.new_vrf1_router_mac }
tests/test_vrf_attr.py:        duthost.options['variable_manager'].extra_vars = extra_vars
tests/test_vrf_attr.py:        extra_vars = { 'router_mac': host_facts['ansible_Ethernet0']['macaddress'] }
tests/test_vrf_attr.py:        duthost.host.options['variable_manager'].extra_vars = extra_vars
```

Comment (wangxin) [reply tests/acl/test_acl.py]: In ansible 2.8.7, extra_vars is exposed as a getter only property. The workaround is to directly access the "private" attribute `_extra_vars`.  So, `extra_vars.clear()` and `extra_vars.update()` won't work. `_extra_vars.clear()` and `_extra_vars.update()` will work.

Comment (msosyak) [reply tests/acl/test_acl.py]: Actually `extra_vars.clear()` and  `extra_vars.update()` will work as `extra_vars` returns referens to the same dict object as `_extra_vars`.  Example: https://onlinegdb.com/BynYjKJ0r

Comment (msosyak) [reply tests/acl/test_acl.py]: @lguohan Fixed. According to https://docs.python.org/3/library/stdtypes.html#dict.update there is no need to call `clear()` method.

**Comment by lguohan** [comment]
thanks!

**Pull request merged**